spec/params/params_spec.rb

```
@@ -172,6 +177,76 @@ module Spec
         Params['tmp5true'].should eq false
         Params['tmp5false'].should eq true
       end
+
+      # If input is not Array we raise exception
+      it 'should raise exception when monitoring_paths format is bad 1' do
+        yml_bad_format="monitoring_paths: '~/.bbfs/backup_files'"
+        expect {
+          Params.read_yml_params(StringIO.new(yml_bad_format))
+          ContentServer.check_monitoring_path_structure('monitoring_paths', 1)
+        }.to raise_error
+      end
+
+      # expecting 1 path for backup_destination_folder
+      # (in call to check_monitoring_path_structure) but user gives
+      # 2 paths.
+      it 'should raise exception when backup_destination_folder format is bad 2' do
+        yml_bad_format=<<EOF
+backup_destination_folder:
+  - path: 'some_path_1'
+    scan_period: 200
+    stable_state: 2
+  - path: 'some_path_2'
+    scan_period: 200
+    stable_state: 2
+EOF
+        expect {
+          Params.read_yml_params(StringIO.new(yml_bad_format))
+          ContentServer.check_monitoring_path_structure('backup_destination_folder', 1)
+        }.to raise_error
+      end
+
+      # missing 'stable_state' in hash
+      it 'should raise exception when monitoring_paths format is bad 3' do

```

Change description to:
should raise exception when missing stable_state parameter